trussworks/react-uswds
# chore: adds shauna to README and updates documentation to reflect usage of all-contributors CLI

# Summary

Re-does adding myself to the contributors file, and improves documentation to prevent similar mistakes from future contributors. In this branch I did three things:

- removes my manually added code from the README file to prevent issues with the CLI tool moving forward
- adds myself using the CLI tool
- updates the `For Trussels` file to instruct new people to use the tool so they don't make the same mistake I did 😅


## Related Issues or PRs

https://github.com/trussworks/react-uswds/pull/2193 - where I added myself manually 

## How To Test

- check if I show up in the README preview
- Review the text I wrote in the `for-trussels.md` file. open to suggestions on how to word it better
